### ISSUE TITLE
Scaled minHash with add_hash_with_abundance

### ISSUE THREAD

**Issue by damiankao** [sourmash-bio/sourmash#2457]
I am using the add_hash_with_abundance method to add pre-calculated kmers with their abundances. Can this work with a scaled minHash? Will the order or kmers streaming in matter?

**Comment by ctb** [comment]
> Can this work with a scaled minHash?

yes! in fact it only works with a scaled MinHash in theory, although I believe our API lets you do it with a regular (num) MinHash.

> Will the order or kmers streaming in matter?

Nope! (Or at least it definitely shouldn't - trust but verify lightly...)

**Comment by ctb** [comment]
> > Will the order or kmers streaming in matter?
> 
> Nope! (Or at least it definitely shouldn't - trust but verify lightly...)

by which I mean do one or two of the most obvious basic tests, nothing big or robust should be needed.
